saleor/graphql/checkout/schema.py

```diff
@@ -51,6 +55,7 @@ class CheckoutQueries(graphene.ObjectType):
             "query checkouts that belong to other users: "
             f"{CheckoutPermissions.MANAGE_CHECKOUTS.name}, "
             f"{AccountPermissions.IMPERSONATE_USER.name}. "
+            f"{PaymentPermissions.HANDLE_PAYMENTS.name}. "
```

Comment:
With your changes, the description is not valid anymore. You can fetch any checkout. Permissions are required only when you want to fetch the checkout from in-active channel.

Reply:
Yes, so should I split the description into two parts - for active and inactive channel or leave it as it has no permission at all?

Reply:
You require permissions for checkouts placed on in-active channels, so I would place this info in the description.